src/main/java/command/CommandProcess.java

```diff
@@ -0,0 +1,120 @@
+package command;
+
+import exception.CommandException;
+import task.*;
+
+import java.util.ArrayList;
```

Comment:
I like this! Use of ArrayList so that the list of tasks do not have a fixed capacity!